### PR TITLE
Do not install bins on system install

### PR DESCRIPTION
**Pull request by zmughal**

### CODE REVIEW

**Comment by zmughal** [comment]
@yoe, @plicease, if you get a chance, could you review this?

File: lib/Dist/Zilla/Plugin/Alien.pm [hunk 1]
```diff
@@ -505,19 +506,30 @@ use strict;
 use warnings;
 use File::ShareDir ':ALL';
 use Path::Class;
-
-my $abs = file(dist_dir('{{ $dist->name }}'),'bin','{{ $bin }}')->cleanup->absolute;
+use File::Which;
```
Comment (plicease) [inline lib/Dist/Zilla/Plugin/Alien.pm:509]: We should add File::Which as a prerequisite when using this option, if not already added.  (gets pulled in already by AB, but we shouldn’t rely on that here).

Comment (zmughal) [reply lib/Dist/Zilla/Plugin/Alien.pm]: Ah, that needs to go away. I removed the code in the else-branch that used `which()`.

**Review by plicease — APPROVED** [review]
I think this is a big improvement, I’d personally like to see the generated installer only generated when we need it, though on the other hand that increases the complexity so it isn’t for free.

File: lib/Dist/Zilla/Plugin/Alien.pm [hunk 2]
```diff
@@ -488,7 +489,7 @@ sub register_prereqs {
 }
 
 has "+mb_class" => (
-	default => 'Alien::Base::ModuleBuild',
+	default => 'MyModuleBuild',
```
[the remark this answers is not in the document]
Comment (zmughal) [reply lib/Dist/Zilla/Plugin/Alien.pm]: Done.

**Pull request merged**